uberenv.py

```diff
@@ -347,6 +348,16 @@ def __init__(self, opts, extra_opts):
         print("[uberenv command line options: ")
         pretty_print_dictionary(self.opts)
         print("]")
+    
+    ###########################
+    # basic spack helpers
+    ###########################
+    def spack_exe_path(self):
+        return pjoin(self.dest_dir,"spack/bin/spack")
+
+    def spack_version(self):
```

Comment:
Were you planning on calling this somewhere?

Reply:
`spack_version`? This was from Cyrus's PR. I wasn't planning on using it in the PR, so I can remove it.